docs/en/FAQ.md

```diff
@@ -1,104 +1,98 @@
+Alluxio requires JVM 1.8 or above to run on various distributions of Linux / MacOS.
 
-3. Is there an easy way to see the status and health of a cluster?
+## What license is Alluxio under?
 
-   Yes, Once Alluxio is installed, the UI is available at `http://master_ip:19999`
+Alluxio is open sourced under the Apache 2.0 license.
 
-4. If I add new nodes to the cluster, will Alluxio re-balance / move the cached blocks to the newly
-added nodes in order to balance memory space utilization between the nodes?
+## My analytics job does not get faster after deploying Alluxio, why?
 
-   No, Currently re-balancing of data blocks in not available.
+There are multiple possible reasons to check out:
+1. The job can be computation bound and does not spend too much time in reading/writing data.
+In this case, speeding up I/O performance will not help too much.
+1. The persistent storage is colocated with compute (e.g., Alluxio is connected to a local
+HDFS), and the input data of the job is buffer-cached.
+1. Due to some mis-configuration, tasks of your job not reading from their local Alluxio workers
+but from network, results in low data-locality.
+1. Input data is already evicted out, the job is not really reading from Alluxio cache.
 
-5. Is Alluxio master fault tolerant?
+## Shall I run Alluxio as a stand-alone system or on YARN, Mesos or Kubernetes?
 
-   Yes. Please see this [page](Running-Alluxio-on-a-Cluster.html#running-alluxio-with-high-availability).
+We recommend to run Alluxio as a stand-alone system.  We support running Alluxio on [YARN]({{
+site.baseurl }}{% link en/deploy/Running-Alluxio-On-Yarn.md %}), [Mesos]({{ site.baseurl }}{% link
+en/deploy/Running-Alluxio-on-Mesos.md %}), and [Kubernetes]({{ site.baseurl }}{% link
+en/deploy/Running-Alluxio-On-Kubernetes.md %}).
 
-6. What are under storage systems and how many under storage systems are supported?
+## Which programming language does Alluxio support?
 
-   Alluxio uses under storage systems as its persistent storage system. It currently supports [Amazon
-   S3](Configuring-Alluxio-with-S3.html), [Swift](Configuring-Alluxio-with-Swift.html), [GCS
-   ](Configuring-Alluxio-with-GCS.html), [HDFS](Configuring-Alluxio-with-HDFS.html) and many more.
+Alluxio is primarily developed in Java and exposes Java-like File APIs for other applications to
+interact with. Alluxio supports other language bindings including [Python client]({{ site.baseurl
+}}{% link en/api/FS-API.md %}#python), [Restful client]({{ site.baseurl }}{% link en/api/FS-API.md
+%}#rest-api).
 
-7. Can I easily add support for other under store systems?
+## What happens if my data set does not fit in memory?
 
-   Yes, in fact support for other under storages is in progress by many contributors. Here is the
-   [documentation](DevelopingUFSExtensions.html) for adding other under storage systems.
+Depends on the system setup, Alluxio may leverage local SSD and HDD. It keeps hot data in Alluxio,
```

Comment:
this sounds like Alluxio might use SSD/HDD automatically. "You can configure Alluxio to leverage SSD and HDD so that your data set does fit in memory"